crm_claim_type/README.md

```diff
@@ -0,0 +1,61 @@
+CRM Claim Types
+==============
+
+Includes a way to clasify claims adding the concept of type.
+It gives a section to define a set of claim types.
+
+Installation
+============
+
+To install this module, you just need to select it from availables modules
+
+Configuration
+=============
+
+This modules do not require special configurations.
+It contains a predefined set of claim types.
+But if you want add your own types go to Sales > Configurations > Claims > Claim Types
+
+Usage
+=====
+
+To use this module, you need to:
+
+* Go to Sales > Configurations > Claims > Claim Types
```

Comment:
Here you have to say that you need to go to claims, and select a type when creating/editing a claim.